src/e3/testsuite/report/rewriting.py

```diff
@@ -28,6 +28,14 @@ class RewritingSummary:
     Set of test names whose result was an error (baseline not updated).
     """
 
+    updated_baselines: set[str]
+    """Set of baselines that were updated.
+
+    These are the baselines that changed, but that were not created nor
+    deleted: they existed before the rewriting, they exist after and their
+    contents are different.
+    """
+
     new_baselines: set[str]
     """Set of test names whose baseline was updated (not empty)."""
```

Comment:
s/updated/created?

Reply:
Right, also the `(not empty)` is misleading (it’s: not empty and was previously not existing): let’s just remove it. Thanks!